homeassistant/components/ihc/__init__.py

```diff
@@ -128,7 +33,7 @@ def setup(hass: HomeAssistant, config: ConfigType) -> bool:
     return True
 
 
-def ihc_setup(hass, config, conf, controller_id):
+def ihc_setup(hass: HomeAssistant, config: ConfigType, conf, controller_id):
```

Comment:
Can you add type hints to the other two parameters?

Reply:
Type hints added, and i renamed conf to controller_conf - to make it more clear, that the second config is the configuration for the individual controller